From 28be1ea9bcba3ea65e49c5605b53b3858fdaa640 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 14 Aug 2019 11:58:26 +0300
Subject: [PATCH] Revert "[SW-0] pcie: gaudi update skip logic"

It turns out this change is not needed, so revert it.

This reverts commit 404c2dc2c9e81d775bf10f0d23d8fdbc64aaa6ff.

Change-Id: I2af1fb09e3c41d3cdde347b7f2c1265054c1bde0
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/asic.c b/gaudi/asic.c
index c760a0d..319331c 100644
--- a/gaudi/asic.c
+++ b/gaudi/asic.c
@@ -297,8 +297,8 @@
 
 u32 hl_skip_pcie_init(void)
 {
-	/* Use DumpSel[11] to skip PCIe initialization (active low) */
-	return !regmap_readl(mmPSOC_GLOBAL_CONF_BOOT_STRAP_PINS) & BIT(19);
+	/* Use DumpSel[11] to skip PCIe initialization */
+	return regmap_readl(mmPSOC_GLOBAL_CONF_BOOT_STRAP_PINS) & BIT(19);
 }
 
 u64 hl_pll_get_cpu_cycles_per_usec(void)
